package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tachyons",
-  "version": "4.7.1",
+  "version": "4.7.2",
```

Comment:
[4.7.2](https://github.com/tachyons-css/tachyons/commit/a791b48bd8dffec70fda0060fc79886e3fad0e2d) is [already a thing](https://www.npmjs.com/package/tachyons). Needs a rebase.